specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Pipeline.json

```diff
@@ -7569,6 +7569,10 @@
           "required": [
             "logDestination"
           ]
+        },
+        "returnMultistatementResult": {
+          "type": "object",
+          "description": "Enable to retrieve result sets from multiple SQL statements and the number of rows affected by the DML statement. Supported connector: SnowflakeV2. Type: boolean (or Expression with resultType boolean)."
```

Comment:
whats the format for this?

Reply:
This is an object that can accept boolean resolved by expressions.

Reply:
This is a data plane change. Seems it's related to [this PR](https://github.com/Azure/azure-rest-api-specs/pull/30014).